upsolver-ui/src/components/App/App.jsx

```diff
@@ -28,6 +29,20 @@ export default function App() {
     localStorage.setItem("user", JSON.stringify(user));
   }, [user]);
 
+  useEffect(() => {
+    socket.auth = {
+      token: user?.token,
+    };
+    socket.connect();
+    socket.on("connect_error", (err) => {
+      console.error("[!] Unable to connect to server", err.message);
```

Comment:
I think it is fine to log these for now, but should there be any communication to the user? Could we use a material messagebox component or something like that? This is just UX considerations but could be nice to handle these other than just a log